[0m19:47:18.866922 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0c3d77a190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0c3d779250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0c3d73eed0>]}


============================== 19:47:18.868928 | 8e77132f-e93b-4bb3-9582-2dd2759811d6 ==============================
[0m19:47:18.868928 [info ] [MainThread]: Running with dbt=1.6.2
[0m19:47:18.869297 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/home/calum/.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m19:47:18.869628 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/calum/eidyn/load/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m19:47:18.870036 [debug] [MainThread]: Command `dbt run` failed at 19:47:18.869954 after 0.01 seconds
[0m19:47:18.870259 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0c3dbfe610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0c3d479ad0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0c41c551d0>]}
[0m19:47:18.870622 [debug] [MainThread]: Flushing usage events
